specification/cognitiveservices/data-plane/Language/preview/2021-07-15-preview/analyzeconversations.json

```diff
@@ -122,21 +129,20 @@
       "type": "object",
       "description": "This is the parameter set of either the conversation application itself or one of the target projects.",
       "required": [
-        "projectType"
+        "targetType"
```

Comment:
"Type" -> "Kind" everywhere.

Reply:
Hi Health. I am trying to understand the term here. Why is "Kind" a better option?